dynawo/3rdParty/CMakeLists.txt

```diff
@@ -0,0 +1,617 @@
+# Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+# See AUTHORS.txt
+# All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+
+# Minimum required (for ExternalProject_Add)
+cmake_minimum_required(VERSION 3.9.6 FATAL_ERROR)
+
+option(BUILD_SHARED_LIBS "Build shared libraries (.dll/.so) instead of static ones (.lib/.a)" ON)
+set(CMAKE_BUILD_TYPE "Release" CACHE STRING "Choose the type of build, options are: Debug Release RelWithDebInfo MinSizeRel")
```

Comment:
Please add a comment to explain the different possibilities as the 2 latest are not obvious.

Reply:
It is default cmake types no need for explanation it can be found in cmake doc

Reply:
Not everyone is a cmake expert so I propose to add a comment "Default cmake build types"

Reply:
I really don't see the point of adding it, if someone tries to modify cmake options (which you should not if you are a simple user) he should know this or a quick search gives all the details on the difference

Reply:
Not sure why adding a comment is such a big deal. I was puzzled by this and so people who will work on the project later might also be.

Reply:
I see that you are against adding a comment and I respect this position so I will accept this cMake file as it is.
However please avoid closing an issue before a proper conclusion was made.